scripts/simulate.js

```diff
@@ -134,19 +148,41 @@ async function fiatLeverage(amount) {
 }
 
 async function leverageCycle(amount) {
+  await network.provider.send("hardhat_setNextBlockBaseFeePerGas",
+    [`0x${Number(40*1e9).toString(16)}`]);
   const signer = (await ethers.getSigners())[0];
 
   const startingEth = await signer.getBalance();
 
   const ptBalance = await purchasePTs(amount);
   const fiatDebt = await collateralizeForFiat();
-  const daiBalance = await curveSwapFiatForDai();
+  const {daiBalance,effectiveFiatPrice,reserves0,reserves1, oldDaiBalance} = await curveSwapFiatForDai();
```

Comment:
since we're already calculating once inside the function may as well pass them up to avoid unnecessary re-calculation.

it doesn't match the other calls nearby, but in a way it makes sense to me, because it's returning information related to the curve swap being executed here.